.gitignore

```diff
@@ -32,3 +32,6 @@
 # Ignore idea files
 .idea
 .env
+
```

Comment:
We do not need application.yml since we already have .env file. The meaning of these two files are the same. So let's store ENV variables in .env file.